srcpkgs/libgaminggear/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'libgaminggear'
+pkgname=libgaminggear
+version=0.15.1
+revision=1
+build_style=cmake
+configure_args="-DINSTALL_CMAKE_MODULESDIR=/usr/share/cmake
+ -DINSTALL_PKGCONFIGDIR=/usr/share/pkgconfig"
```

Comment:
should be /usr/lib since this is a library, the results will vary from architecture to architecture.

same for CMAKE_MODULESDIR